specification/mediaservices/resource-manager/Microsoft.Media/stable/2020-05-01/examples/liveevent-delete.json

```diff
@@ -16,7 +16,7 @@
           "tag2": "value2"
         }
       },
-      "properties": {
+      "Properties": {
```

Comment:
Are these case changes considered breaking in any way? Will they break any SDKs generated?

Reply:
Yes, breaking

Reply:
all of these looks like issues with examples not the actual swagger. let me work with the service team to fix these.

---
In reply to: [497809167](https://github.com/Azure/azure-rest-api-specs/pull/10960#discussion_r497809167) [](ancestors = 497809167)